src/main/java/org/opencds/cqf/tooling/terminology/CodeSystemLookupDictionary.java

```diff
@@ -61,7 +62,7 @@ public class CodeSystemLookupDictionary {
         UBTOB                       2.16.840.1.113883.6.21          ->      http://terminology.hl7.org/CodeSystem-nubc-UB92.html
         POS                         2.16.840.1.113883.6.50          ->      http://terminology.hl7.org/CodeSystem-POS.html
         PYXIS                       ???                             ->      ???
-        CDCREC                      2.16.840.1.113883.6.238         ->      http://terminology.hl7.org/CodeSystem-PHRaceAndEthnicityCDC.html
+        CDCREC                      2.16.840.1.113883.6.238         ->      https://terminology.hl7.org/3.1.0/NamingSystem-CDCREC.html
```

Comment:
The value here is the link to the CDCREC naming system, I don't think that's correct.
According to THO, the URL for this codesystem should be: urn:oid:2.16.840.1.113883.6.238 (Source: https://terminology.hl7.org/CodeSystem-CDCREC.html)